### ISSUE TITLE
master nodes should merge first

### ISSUE THREAD

**Issue by yegor256** [zold-io/zold#645]
Let's merge master nodes first.

**Comment by 0crat** [comment]
The job [#645](https://github.com/zold-io/zold/issues/645) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +30 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)
